Makefile.include

```diff
@@ -110,6 +112,20 @@ gen-alertmanager:     # Generate Alertmanager client.
 clean_swagger:
 	find api -name '*.swagger.json' -print -delete
 
+LATEST_SWAGGER = $(shell curl --silent --head https://github.com/swagger-api/swagger-ui/releases/latest | grep -E '^[lL]ocation:' | grep -oE 'v[0-9.]+[[:space:]]*' | tr -d ' ')
```

Comment:
Dude, you are a bash ninja :)

Reply:
🚢

Reply:
Let's wait until it's tested, OK?

Reply:
Minor: this variable can be moved under first `if` in `download` swagger. It's used only there.